.github/workflows/build.yaml

```diff
@@ -846,7 +846,7 @@ jobs:
           dnf config-manager --set-enabled powertools
           dnf install --setopt=tsflags=nodocs --setopt=install_weak_deps=False -y epel-release 'dnf-command(copr)' 'dnf-command(builddep)'
           dnf copr enable -y clime/rpkg-util
-          dnf install --setopt=tsflags=nodocs --setopt=install_weak_deps=False -y git rpkg
+          dnf install --setopt=tsflags=nodocs --setopt=install_weak_deps=False -y git rpkg libunwind-devel
```

Comment:
`libunwind-devel` here is redundant, it is installed by `dnf builddep` when it scans the spec file.

* I'll fix it in https://github.com/skupperproject/skupper-router/pull/1120